includes/class-wc-gateway-ppec-client.php

```diff
@@ -458,72 +458,7 @@ protected function _get_details_from_cart() {
 			'items'             => $this->_get_paypal_line_items_from_cart(),
 		);
```

Comment:
So the only two things I can see is per our standardization spec we no longer prefix private/protected methods with `_` underscore. And the other thing perhaps is if we need to cast anything to `float` in the calculations in case PHP7 barks. Just a mention as it may or may not be necessary in your case.